doc/user/methods/gradient_boosting.md

```diff
@@ -0,0 +1,241 @@
+   - Train on mixed categorical data using the `minimumLeafSize`, `minimumGainSplit`, and `maximumDepth` options to control the behavior of the weak learners ([`DecisionTreeRegressor`s](decision_tree_regressor.md)).
+
+---
+
+Types of each argument are the same as in the table for constructors
+[above](#constructor-parameters).
+
+***Note:***: Training is not incremental.  A second call to `Train()` will retrain the
+model from scratch.
+
+### Classification
+
+Once a `GradBoosting` object is trained, the `Classify()` member function can be used
+to make class predictions for new data.
+
+ * `size_t predictedClass = gb.Classify(point)`
+    - ***(Single-point)***
+    - Classify a single point, returning the predicted class.
+
+---
+
+ * `gb.Classify(point, prediction)`
+    - ***(Single-point)***
+    - Classify a single point and compute class probabilities.
+    - The predicted class is stored in `prediction`.
+
+---
+
+ * `gb.Classify(data, predictedLabels)`
+    - ***(Multi-point)***
+    - Classify a set of points.
+    - The prediction for data point `i` can be accessed with `predictedLabels[i]`.
+
+---
+
+#### Classification Parameters:
+
+| **usage** | **name** | **type** | **description** |
+|-----------|----------|----------|-----------------|
+| _single-point_ | `point` | [`arma::vec`](../matrices.md) | Single point for classification. |
+| _single-point_ | `prediction` | `size_t&` | `size_t` to store class prediction into. |
+||||
+| _multi-point_ | `data` | [`arma::mat`](../matrices.md) | Set of [column-major](../matrices.md#representing-data-in-mlpack) points for classification. |
+| _multi-point_ | `predictedLabels` | [`arma::Row<size_t>&`](../matrices.md) | Vector of `size_t`s to store class prediction into.  Will be set to length `data.n_cols`. |
+
+***Note:*** different types can be used for `data` and `point` (e.g.
+`arma::fmat`, `arma::sp_mat`, `arma::sp_vec`, etc.).  However, the element type
+that is used should be the same type that was used for training.
+
+### Other Functionality
+
+ * `WeakLearner(size_t i)` returns a [`DecisionTreeRegressor&`]
+    (decision_tree_regressor.md) indicating which weak learner 
+    is being used. 
+
+ * `gb.NumWeakLearners()` returns a `size_t` indicating how 
+    many weak learners are being used by the model.
+    - `gb.NumWeakLearners(x)` will set the number of weak learners 
+    that will be used the *next time* `Train()` is called to `x`.
+
+ * `gb.NumClasses()` returns a `size_t` indicating the number of classes the
+   model was trained on.
+
+For complete functionality, the [source
+code](/src/mlpack/methods/grad_boosting/grad_boosting.hpp) can be consulted.
+Each method is fully documented.
+
+### Simple Examples
+
+See also the [simple usage example](#simple-usage-example) for a trivial use of
+`GradBoosting`.
+
+---
+
+Train a gradient boosting model on mixed categorical data, passing 
+arguments into the Train function:
```

Comment:
```suggestion
Train a gradient boosting model on mixed categorical data with the
`Train()` function:
```
(I think this is just a little bit of a simpler way to put it.  Up to you if you want to accept the suggestion.)